package.json

```diff
@@ -19,7 +19,7 @@
     "test:e2e": "node tests/e2e/local-run.js",
     "test:e2e:build": "cross-env VUE_APP_E2E=true vue-cli-service build --mode production",
     "test:e2e:browserstack": "node tests/e2e/browserstack.runner.js -c  tests/e2e/browserstack.conf.js",
-    "test:e2e:local": "kill-port 9515 && nightwatch --config ./tests/e2e/local-chrome.conf.js",
+    "test:e2e:local": "nightwatch --config ./tests/e2e/local-chrome.conf.js",
```

Comment:
why did you remove this?

Reply:
Sorry, my fault. kill-port always needs more than 3 minutes to be executed, so that's why i'm removing it always. Forgot to add it back

Reply:
on Windows. Maybe there is a better plugin.

Reply:
https://github.com/tiaanduplessis/kill-port/pull/39 :/